02-start.Rmd

```diff
@@ -76,7 +76,11 @@ bioc_pkg <- c(bioc_pkg[ bioc_pkg %in% c("mia", "miaViz") ], bioc_pkg[ !bioc_pkg
 # Loading all packages into session. Returns true if package was successfully loaded.
 sapply(c(bioc_pkg, cran_pkg), require, character.only = TRUE)
 ```
- 
+
+```{r}
+# Setting path for saving figures generated throughout the notebook.
+knitr::opts_chunk$set(echo = FALSE, fig.path="Figures/")
+```
 
```

Comment:
Cool! Is it possible to move that to main.R file, or is there any restrictions for that? I suggest that it could be better place for this, because it already includes some settings for the book

Reply:
Well, then `knitr` package might not be installed, so I think this is ok

Reply:
I'll check and give it a try

Reply:
> Well, then `knitr` package might not be installed, so I think this is ok
ok

Reply:
I think it is possible to set global options that way. Check it at least.

Reply:
Ok, I managed to move it to main.R